src/core/MOM_verticalGrid.F90

```diff
@@ -186,18 +190,22 @@ subroutine verticalGridInit( param_file, GV, US )
     GV%m_to_H = 1.0 / GV%H_to_m
     GV%H_to_MKS = GV%H_to_m
     GV%m2_s_to_HZ_T = GV%m_to_H * US%m_to_Z * US%T_to_s
+
+    GV%H_to_Z = GV%H_to_m * US%m_to_Z
+    GV%Z_to_H = US%Z_to_m * GV%m_to_H
   else
     GV%kg_m2_to_H = 1.0 / GV%H_to_kg_m2
-    GV%m_to_H = US%R_to_kg_m3*GV%Rho0 * GV%kg_m2_to_H
-    GV%H_to_m = GV%H_to_kg_m2 / (US%R_to_kg_m3*GV%Rho0)
+    !  GV%m_to_H = US%R_to_kg_m3*GV%Rho0 * GV%kg_m2_to_H
```

Comment:
Did you mean to keep this line?

Reply:
Leaving this comment was intentional.  It is a reminder that when all this is done, we might want to revert to using RHO_0 directly for one of the 8 parameters for which it the default (most likely this one).  However, this might not be a necessary reminder, if we would prefer to remove it.